lib/core/src/Cardano/Wallet/DB/Checkpoints/Model.hs

```diff
@@ -74,23 +91,26 @@ is clear that the data cannot exist at the genesis point
 -}
 
 {-------------------------------------------------------------------------------
-    Checkpoints
+        Checkpoints
 -------------------------------------------------------------------------------}
+
 -- | Collection of checkpoints indexed by 'Slot'.
 newtype Checkpoints a = Checkpoints
-    { checkpoints :: Map W.Slot a
-    -- ^ Map of checkpoints. Always contains the genesis checkpoint.
-    } deriving (Eq,Show,Generic)
+    { -- | Map of checkpoints. Always contains the genesis checkpoint.
+        checkpoints :: Map W.Slot a
```

Comment:
Hi @paolino 

Is there any reason to change the style of haddock comment here? It seemed fine before.

I think this kind of change should be avoided, _unless_ the code was breaking our coding standard in some way.

In the wallet we generally use the style where the haddock comment for a record field follows the field itself:

```suggestion
    { checkpoints :: Map W.Slot a
    -- ^ Map of checkpoints. Always contains the genesis checkpoint.
```
In some cases (typically for larger records with long comments), we indent each comment by one block (4 spaces):
```suggestion
    { checkpoints :: Map W.Slot a
        -- ^ Map of checkpoints. Always contains the genesis checkpoint.
```
Would it be possible to go through this PR and make sure that all changed/added record fields are of this form?

Many thanks!